bin/pycbc_single_template

```diff
@@ -293,8 +295,18 @@ with ctx:
     parse_row.params = t()
     for param in row.fieldnames:
         setattr(parse_row.params, param, row[param][0])
-    chisq_bins = int(vetoes.SingleDetPowerChisq.parse_option(parse_row,
-                                                               opt.chisq_bins))
+
+    try:
+        chisq_bins = int(vetoes.SingleDetPowerChisq.parse_option(parse_row,
+            opt.chisq_bins))
+    except ValueError:
+        if ops.default_chisq_bins:
+            chisq_bins = ops.default_chisq_bins
+            warnings.warn(
```

Comment:
done in https://github.com/ligo-cbc/pycbc/pull/1219/commits/31205f7640f0a1cc7ffd4d9974b7bff20864f399